mappings/net/minecraft/world/GameRuleDispatcher.mapping

```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/class_5081 net/minecraft/world/GameRuleDispatcher
```

Comment:
GameRuleManager

Reply:
already did in 98d0ad7
reload the page :tiny_potato: